cli/exchange/agbot.go

```diff
@@ -133,6 +133,8 @@ func AgbotAddBusinessPolicy(org, userPw, agbot, polOrg string) {
 	org, agbot = cliutils.TrimOrg(org, agbot)
 
 	input := exchange.ServedBusinessPolicy{BusinessPolOrg: polOrg, BusinessPol: "*", NodeOrg: polOrg}
+        i18n.GetMessagePrinter().Printf("Adding Business policy org %s' to agbot '%s' ...", polOrg, agbot)
```

Comment:
"Adding Business policy org %s' to agbot '%s'. The agbot will start looking for nodes that are compatible with this policy."